apps/api/src/routes/v1_ratelimit_listOverrides.ts

```diff
@@ -0,0 +1,121 @@
+import { rootKeyAuth } from "@/pkg/auth/root_key";
+import { openApiErrorResponses } from "@/pkg/errors";
+import type { App } from "@/pkg/hono/app";
+import { createRoute, z } from "@hono/zod-openapi";
+import { buildUnkeyQuery } from "@unkey/rbac";
+
+const route = createRoute({
+  tags: ["ratelimit"],
+  operationId: "listOverrides",
+  method: "get",
+  path: "/v1/ratelimit.listOverrides",
+  security: [{ bearerAuth: [] }],
+  request: {
+    query: z.object({
+      namespaceId: z.string().openapi({
+        description:
+          "Namespaces group different limits together for better analytics. You might have a namespace for your public API and one for internal tRPC routes.",
+        example: "email.outbound",
+      }),
+      identifier: z.string().optional().openapi({
+        description:
+          "Identifier of your user, this can be their userId, an email, an ip or anything else.",
+        example: "user_123",
+      }),
+    }),
+  },
+  responses: {
+    200: {
+      description: "List of keys for the api",
+      content: {
+        "application/json": {
+          schema: z.object({
+            overrides: z.array(
+              z.object({
+                id: z.string(),
+                namespace: z.string(),
```

Comment:
probably overkill to return the namespace name, they already queried for it.

Reply:
Sounds good removed